### ISSUE TITLE
Taking a close look at CRS3 performance against WAVSEP

### ISSUE THREAD

**Issue by dune73** [SpiderLabs/owasp-modsecurity-crs#685]
The Web Application Vulnerability Scanner Evaluation Project at 
https://github.com/sectooladdict/wavsep/
seems to be the tool of choice to determine the quality of a scanner. This due to the fact that the density of vulnerabilities is very high.

I am currently looking into this together with a friend. Creating this as an issue and adding labels, even if they do not really fit.

**Comment by csanders-git** [comment]
@dune73 how is this going?

**Comment by dune73** [comment]
The test server is configured, testing is about to start.

**Comment by dune73** [comment]
Setup description: https://www.netnea.com/cms/2017/02/24/optimized-lab-setup-to-attack-modsecurity-with-security-sanners/

**Comment by csanders-git** [comment]
I like how you changed the paranoia level as part of a request... perhaps we should add this to the regression tests

**Comment by dune73** [comment]
It's quite neat, thanks. There is also a session-tracking field in the logformat that I advocate. Setting this via another X-header lets us group it all very easily. Btw. burp scanner had a detection rate of 97% in PL4. What is open can mostly not be considered an attack in itself. (Reconnaissance or specific weaknesses in the backend triggered by certain innocent requests)

**Comment by csanders-git** [comment]
@dune73 wow, that's great! i'm glad this project is being undertaken formally

**Comment by sathieu** [comment]
Hello, Any news on this?

**Comment by dune73** [comment]
No, not really. My research partner dropped out and he never published the results of the part that we finished. However, I have the data for burp (at hand!) and I stand by the data behind one slide that I use in my presentations: https://de.slideshare.net/ChristianFolini/introducing-the-owasp-modsecurity-core-rule-set-75913263, slide number 9.

I expect the results of the other scanners to be similar, and it's a pity we did not finish this nor formally publish.
